tern/command_lib/base.yml

```diff
@@ -379,3 +379,41 @@ npm:
           - "pkgs=`npm list -g | sed 's/\\(^[^a-z]*\\)\\([_.a-z0-9-]*\\)\\(@.*\\)/\\2/g' | grep -v '/usr*' | uniq`"
           - "for p in $pkgs; do npm view $p repository.url; done"
     delimiter: "\n"
+# golang----------------------------------------------------------------------
+go:
+  pkg_format: ''
+  os_guess:
+    - 'None'
+  path:
+    - 'usr/local/go/bin'
+  pkg_separators:
+    - '@'
+    - '@^'
+    - '@~'
+  pinning_separator: ':'
+  names:
+    invoke:
+      1:  
+        container:
+          - "/usr/local/go/bin/go list -m all | tail -n +2 | cut -d ' ' -f 1" 
+    delimiter: "\n"
+  versions:
+    invoke:
+      1:
+        container:
+          - "/usr/local/go/bin/go list -m all | tail -n +2 | cut -d ' ' -f 2"
+    delimiter: "\n"
+  copyrights:
+    invoke:
+      1:
+        container:
+          - "export GOPATH=/go"
+          - "dirs=`/usr/local/go/bin/go list -m -f={{{{.Dir}}}} all | tail -n+2`"
+          - "for d in $dirs;do find $d -type f -iname 'License' -o -iname 'license.txt' -o -iname 'license.md' | xargs cat; echo LICF ; done;"
```

Comment:
small nit: semi-colons should hug the character before and there should be a space after the semi-colon before the proceeding character. This line needs a few fixes -- 1) `for d in $dirs; do` and 2) `echo LICF; done`